index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>JD's Portfolio</title>
+</head>
+<body>
+    <div class="index-page">
+        <header>        
+        <nav class="flex-navbar">
+            <div>
+                <a href="#">
+                    <img class="primary-logo" src="img/archesportslogo.jpg" alt="ArchEsportsLogo">
+                </a>
+            </div>            
+            <ul class="flex-nav-list">
+               <li class="list-item"><a href="#">Portfolio</a></li>
+               <li class="list-item"><a href="#">About</a></li>
+               <li class="list-item"><a href="#"> Contact </a></li>
+            </ul>
+            <div class="menu">
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+            </div>
+        </nav>
+        </header>
+        <section id="headline-section">
+            <h4 class="headline-title"> Hi there!👋🏼
+                I'm JD 🤭</h4>
```

Comment:
- [ ] Well done on correctly identifying the title of your project :+1:. However, to ensure semantic accuracy, it is recommended that the main title of your page should be marked up with an `h1` tag instead of an `h4` tag. This is because the `h1` tag carries more weight in terms of search engine optimization and assists in organizing the structure of your page for both users and web crawlers. Using an `h1` tag for the main title will help to improve the accessibility and usability of your webpage :+1: .